website/docs/utilities/popover-primitive/partials/code/component-api.md

```diff
@@ -0,0 +1,123 @@
+  </C.Property>
+  <C.Property @name="enableSoftEvents" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Assigns "soft" events listeners (`mouseEnter/Leave` + `focusIn/Out`) to the toggle, to control the visibility of the popover content.
+  </C.Property>
+  <C.Property @name="enableClickEvents" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Assigns a "click" event listener (`onClick`) to the toggle, to control the visibility of the popover content.
+  </C.Property>
+  <C.Property @name="onOpen" @type="function">
+    A callback function invoked when the popover is opened (if provided).
+  </C.Property>
+  <C.Property @name="onClose" @type="function">
+    Provides a callback function invoked when the popover is closed (if provided).
+  </C.Property>
+  <C.Property @name="[PP].setupPrimitiveContainer" @type="modifier">
+    Provides a modifiers that needs to be applied to the container of the toggle and popover elements.
+  </C.Property>
+  <C.Property @name="[PP].setupPrimitiveToggle" @type="modifier">
+    Provides a modifiers that needs to be applied to the toggle element.
+    <br />
+    <br />
+    _⚠️ Important: the HTML element must be a `<button>` for accessibility conformance. If not, the component will throw an errow._
+  </C.Property>
+  <C.Property @name="[PP].setupPrimitivePopover" @type="modifier">
+    Provides a modifiers that needs to be applied to the popover element. It accepts an `anchoredPositionOptions` object as "named" argument, with different keys corresponding to different options (see below). These options are forwarded to underlying modifier `hds-anchored-position`, and in turn are passed down to the [Floating UI](https://floating-ui.com) library.
+    <Doc::ComponentApi as |C|>
+      <C.Property @name="anchoredPositionOptions.placement" @type="enum" @values={{array "top" "top-start" "top-end" "right" "right-start" "right-end" "bottom" "bottom-start" "bottom-end" "left" "left-start" "left-end"}} @default="bottom">
+        Placement for the preferred starting position of the popover relative to the toggle element.
+        <br />
+        <br />
+        _Notice: if `@enableCollision` is set, the popover will automatically shift position to remain visible when near the edges of the screen regardless of the starting placement._
+      </C.Property>
+      <C.Property @name="anchoredPositionOptions.strategy" @type="enum" @values={{array "absolute" "fixed"}} @default="absolute">
+        Controls how the layout of the popover is applied using the CSS `position` property. Since the component uses the [native web Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API) which promotes the popover to the [top layer](https://developer.mozilla.org/en-US/docs/Glossary/Top_layer), there is no need to use the `fixed` position to avoid stacking context conflicts, but we leave the option open for edge cases in which this is needed anyway.
```

Comment:
```suggestion
        Controls how the layout of the popover is applied using the CSS `position` property. Since the component uses the [native web Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API) which promotes the popover to the [top layer](https://developer.mozilla.org/en-US/docs/Glossary/Top_layer), there is no need to use the `fixed` position to avoid stacking context conflicts, but we leave the option open for edge cases in which this is needed.
```